src/main/java/net/openhft/chronicle/bytes/BytesStore.java

```diff
@@ -397,12 +397,19 @@ default char charAt(int index)
     }
 
     /**
-     * Not supported.
+     * This method is inherited from CharSequence.
```

Comment:
I think we should not add a JavaDoc here as it hides the original one from CharSequence. Maybe put it as a comment instead.

Reply:
Personally I like 
```
/**
 * {@inheritDoc}
 */
```
in cases like this.

However, in this particular case I find the comment useful. At least, if there were only inherited doc, I'd check which BytesStore type is constructed.